scripts/common/get_logs.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+__author__="Amandeep Singh Arora"
+__version__="1.0"
+
+> /tmp/fledge_syslog.log
+
+exec 2<&-
+exec 2<>/tmp/fledge_syslog.log
+
+RECALC_AFTER_N_SCRIPT_RUNS=100
+NUM_LOGFILE_LINES_TO_CHECK_INITIALLY=2000
+
+offset=100
+limit=5
+pattern=""
+level="debug"
+logfile="/var/log/syslog"
+sourceApp="fledge"
```

Comment:
It's better to add comments what these variables are used for.

Reply:
Their meaning is pretty obvious if you look around a bit in the script.